### ISSUE TITLE
Steps for deprecation of the intermagnet.org website to intermagnet.github.io

### ISSUE THREAD

**Issue by CharlesBlais** [INTERMAGNET/wg-www-gins-data-formats#8]
Hi all,

As some of you may have read from the presentation, INTERMAGNET.org will have to be deprecated in the near future.  Some works as already been done (like putting a notice on the website) and here is the plan for the remaining items.

1. Data Formats pages

Alter the content of the page to reference to sections of the technical manual (page #) until the technical manual is made into a web friendly format.  Once converted, change the reference to actual links (with anchor) to the sections of the web version.

2. Observatory plots

Create a page indicating the observatory plot is under construction.  In its current state, it can not be hosted on github.io since it is a server side library (IDL) can github.io can only handle client side libraries (javascript).  It will eventually be hosted by BGS with their new application.

3. Geomagnetic activity maps

This one is harder.  This application is two sections.  It calculates hourly ranges of all observatories each 15 minutes and then generates images using (IDL) which neither can be hosted on github.io.  I suggest, deprecating it entirely until another institute is willing to do both hourly ranges and an interactive map application on github.  Any other ideas?

4. Photographs and yearbooks

Github can hold the large yearbooks and possibly the photographs.  We could try and put the photographs on this but is it really required?  For yearbooks, could another institute host this on their website or FTP?  Any other ideas?  Do we keep hosting yearbooks?

5. Participating institutes

This could possibly be added to the meta application but not sure.  I don't remember what I did for that application.

6. Technical Manual

Already discussed with Benoit and it will be moved as PDF on github.  I will help Benoit changing the links to point to the PDF.

Any inputs welcomed!

**Comment by ALewis-GA** [comment]
4. All the yearbooks must be already hosted on IMO/institute web pages - could static links to those sites serve the purpose? 

 I don't think the IMO photographs are necessary.

Looks like you have already developed a  Map of IMOs (on the GitHub metadata application page) to replace https://www.intermagnet.org/imos/imomap-eng.php

**Comment by vmaury-ipgp** [comment]
Andrew you are right, the yearbook are on the BCMT website :
http://www.bcmt.fr/bulletins.html

We do have as well description and photos of our IMOs for all our Network. 
http://www.bcmt.fr/networks2.html

We could restrict to Intermagnet Network. There is already an "Intermagnet web page" that we can adapt.

**Comment by CharlesBlais** [comment]
#5 list institutes completed and published at https://intermagnet.github.io/metadata/

**Comment by SimonFlower** [comment]
My thoughts on your numbered items:

1. Data Formats pages

Yes - agree this is a way forward. I know there's already been discussion on this in the technical manual subcommittee, so it would be good to make contact with them. Perhaps we should have a shared meeting on web delivery of the technical manual?

2. Observatory plots

We already have pages for viewing data in the Edinburgh GIN. They aren't new (at all)! In fact they look rather dated. But these could be used now (if I remove the password protection from them). They wouldn't allow access to all data until we sort out the transfer of the archive, but they might be better than no access at all?

3. Geomagnetic activity maps

Again, we have the web service at BGS (giving access to the data). Is another institute able to work on a generating the range data and display?

4. Photographs and yearbooks

No further thoughts

5. Participating institutes

Institutes are already recorded as part of the metadata database that we host at BGS. It will be simple to write a web service to access the institute data. Can someone else look at a client-side application to display that data?

6. Technical Manual

No further thoughts

**Comment by CharlesBlais** [comment]
Yes, #5 is done as per comment earlier.  I added another page to the client side app using your metadata.  Up for people to review if they wish.

**Comment by SimonFlower** [comment]
At the end of the switchover between websites can we reassign the DNS entry for intermagnet.org to point to intermagnet.github.io, or is it more complicated than that? I know I've seen instructions somewhere on custom domains and GitHub...

**Comment by CharlesBlais** [comment]
It seems its doable by getting the DNS entry modified with a CNAME that points to intermagnet.github.io.  This will however mean that Canada still controls DNS and the domain.  I think this is ok.  I don't, however, hold control over our DNS (its a central service) so we might to split this off into two steps.

1. Do a page with link and possibly a redirect (not sure how HTTPS permits redirect from one SSL to another SSL).
2. Have our central service change the A record to a CNAME record to intermagnet.github.io

**Comment by teraita** [comment]
3. Our director (SGO) pointed two person from UOulu/Aalto, who are working in her projects. They can investigate, if they are able to the regenerating the geomagnetic maps. They are already working on new apps from available geomagnetic data.

**Comment by stephanbracke** [comment]
Charles is it possible to reuse the domain name intermagnet.org instead of intermagnet.github.io 
I read about this possibility in github.io [here](https://docs.github.com/en/github/working-with-github-pages/configuring-a-custom-domain-for-your-github-pages-site) If so that would be nice to have a smooth transition

**Comment by teraita** [comment]
Last spring we redirected SGO old data server to the  main website, when it was closed. http://sgodata.sgo.fi goes directly to https://www.sgo.fi. The https://sgodata.sgo.fi warns, but goes trough to the https://www.sgo.fi. So I think it is doable.

**Comment by CharlesBlais** [comment]
Yes, its doable as per my earlier comment in this thread.

**Comment by CharlesBlais** [comment]
As for @teraita regarding having SGO doing dev work for an activity map, that is great.  We'll wait for BGS and NRCan to complete the transition and then I'll let @SimonFlower, if he can, to follow up with you when he thinks his best.
